VideoWeb/VideoWeb/ClientApp/src/app/shared/models/hearing.ts

```diff
@@ -59,8 +59,7 @@ export class Hearing extends HearingBase {
     getEndpoints(): VideoEndpointResponse[] {
         if (this.conference instanceof ConferenceResponse) {
             const conf = this.conference as ConferenceResponse;
-            const ep = conf.endpoints ? conf.endpoints : new Array<VideoEndpointResponse>();
-            return ep;
+            return conf.endpoints ? conf.endpoints : new Array<VideoEndpointResponse>();
```

Comment:
you can use || here instead

Reply:
This wasn't anything to do with this PR but was done to satisfy a complaint of SonarQube.
Code now changed to your suggestion.